core/controllers/admin_test.py

```diff
@@ -63,8 +63,14 @@
 BOTH_MODERATOR_AND_ADMIN_EMAIL = 'moderator.and.admin@example.com'
 BOTH_MODERATOR_AND_ADMIN_USERNAME = 'moderatorandadm1n'
 
-PARAM_NAMES = python_utils.create_enum('test_feature_1')  # pylint: disable=invalid-name
-FEATURE_STAGES = platform_parameter_domain.FEATURE_STAGES
+
+class PARAM_NAMES(enum.Enum): # pylint: disable=invalid-name
+    """Enum for parameter names."""
+
+    test_feature_1 = 'test_feature_1' # pylint: disable=invalid-name
+
+
+FEATURE_STAGES = platform_parameter_domain.FEATURE_STAGES # pylint: disable=invalid-name
```

Comment:
can you please suggest me any alternative name for 
`from core.domain import app_feedback_report_constants as constants`
because directly using app_feedback_report_constants 80-max length lint problem.